filepicker/src/main/java/dev/arkbuilders/components/filepicker/FragmentManagerUtils.kt

```diff
@@ -20,4 +20,21 @@ fun FragmentManager.onArkPathPicked(
             )
         )
     }
+}
+
+fun FragmentManager.onArkRootOrFavPicked(
+    lifecycleOwner: LifecycleOwner,
+    listener: (path: Path, rootNotFavorite: Boolean) -> Unit
+) {
+    setFragmentResultListener(
+        ArkRootPickerFragment.ROOT_PICKED_REQUEST_KEY,
+        lifecycleOwner
+    ) { _, bundle ->
+        listener(
+            Path(
+                bundle.getString(ArkRootPickerFragment.PICKED_PATH_BUNDLE_KEY)!!
```

Comment:
Instead of asserting the returned string value, what about giving it a non-null fallback value?
It will never crash the app even if the value cannot be found from this bundle.

```suggestion
               bundle.getString(ArkFilePickerFragment.PATH_PICKED_PATH_BUNDLE_KEY, "")
```

Reply:
This is key for selected path from dialog. If we return fragment result bundle without selected path, then something goes terribly wrong. We want to crash in this case.